.azure/scripts/wheels.yml

```diff
@@ -3,12 +3,15 @@ steps:
 - script: |
     mkdir -p dist
     python -m pip install --upgrade pip
-    python -m pip install --upgrade wheel setuptools
   displayName: 'Install dependencies'
 
 - script: |
     python -m pip wheel . -w wheelhouse/
   displayName: 'Build wheel'
+  inputs:
```

Comment:
Sadly this is only done on release. It would be good to validate against a wheel in our systematic testing.

Reply:
In the previous system we used setupext to trigger a custom logics in the sdist which causes the jar to be built and included.  Without this logic it may just create a bogus sdist package.

We don't build the jar on the users system because it causes way too many issues when people have jar built with random Java versions.   Some builds with a later versions then distributes it to another user with an earlier version and we get version errors that weren't tied to anything in our release.   Thus jar must be shipped.